docs/docs/examples/bitbucket/BITBUCKET_MESH.md

```diff
@@ -0,0 +1,78 @@
+# Bitbucket Mesh
+
+Bitbucket Mesh is a distributed, replicated, and horizontally scalable Git repository storage system, which increases performance and improves the resilience of Bitbucket.
+
+![bitbucket_mesh](../../assets/images/bitbucket_with_mesh.png)
+
+You can learn more details about Bitbucket Mesh on the [official documentation page](https://confluence.atlassian.com/bitbucketserver/bitbucket-mesh-1128304351.html){.external}.
+
+!!!tip "Recommendations for Mesh deployments"
+
+    **Bitbucket version**
+
+    By default the Helm charts target the latest Bitbucket LTS version. However
+    mesh is only supported from version 8.0. You will need to select an 8.x
+    version of Bitbucket to deploy Mesh. See below for details.
+
+    **Mesh agent version**
+
+    Bitbucket Mesh agents are versioned independently from Bitbucket. You should
+    select the appropriate version for the deployed version of Bitbucket. See
+    the [Mesh download page](https://www.atlassian.com/software/bitbucket/download-mesh-archives){.external}
+    for available versions.
+
+    **Number of Mesh nodes**
+
+    In order for high-availability to be possible, we recommend having a minimum
+    of three Mesh nodes. There is no maximum on the number of nodes.
+
+    **Mesh node co-location**
+
+    Currently, we will not be supporting deploying Mesh nodes into multiple
+    availability zones. Just like the shared file system based deployments, the
+    Mesh nodes (that is, the repository storage) and the application nodes must
+    be co-located.
+
+    **Other Mesh deployment requirements**
+
+    For more details on the requirements and limitations of Mesh deployments see
```

Comment:
For more details on the requirements and limitations of Mesh deployments, [check the Bitbucket Mesh FAQ](https://confluence.atlassian.com/enterprise/bitbucket-data-center-faq-776663707.html#BitbucketDataCenterFAQ-Mesh).